
string device_id
int32 model_id
string model_name
int32 manufacturer_id
string manufacturer_name
string hardware_version
string software_version

string sdp_version
string sdk_version
